apps/web/prisma/schema.prisma

```diff
@@ -278,3 +280,23 @@ model Campaign {
 
   team Team @relation(fields: [teamId], references: [id], onDelete: Cascade)
 }
+
+model ApiLog {
+  id          String   @id @default(cuid())
+  teamId      Int
+  method      String
+  path        String
+  status      Int
+  duration    Int // in milliseconds
+  apiKeyId    Int?
+  requestBody Json? // Store request body
```

Comment:
let's just use string to have better performance. Json seems to be overkill here.

Reply:
Converted to string field as suggested